<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsiderationDialog</class>
 <widget class="QDialog" name="ConsiderationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>検討</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <item>
    <widget class="QGroupBox" name="groupBoxEngine">
     <property name="title">
      <string>エンジン設定</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutEngine">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutEngine">
        <item>
         <widget class="QComboBox" name="comboBoxEngine1">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="engineSetting">
          <property name="text">
           <string>エンジン設定</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerEngine">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTime">
     <property name="title">
      <string>思考時間</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutTime">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutUnlimited">
        <item>
         <widget class="QRadioButton" name="unlimitedTimeRadioButton">
          <property name="text">
           <string>時間無制限</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerUnlimited">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutByoyomi">
        <item>
         <widget class="QRadioButton" name="considerationTimeRadioButton">
          <property name="text">
           <string>検討時間</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="byoyomiSec">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>999999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>秒まで</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerByoyomi">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMultiPV">
     <property name="title">
      <string>候補手</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutMultiPV">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutMultiPV">
        <item>
         <widget class="QLabel" name="labelMultiPV">
          <property name="text">
           <string>候補手の数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxMultiPV">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="toolTip">
           <string>評価値が大きい順に表示する候補手の数を指定します</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMultiPVUnit">
          <property name="text">
           <string>手</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerMultiPV">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutBottom">
     <item>
      <widget class="QToolButton" name="toolButtonFontDecrease">
       <property name="text">
        <string>A-</string>
       </property>
       <property name="toolTip">
        <string>フォントサイズを小さくする</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonFontIncrease">
       <property name="text">
        <string>A+</string>
       </property>
       <property name="toolTip">
        <string>フォントサイズを大きくする</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerBottom">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
